### ISSUE TITLE
ProxyAutoConfigUrl for ProxyKind.ProxyAutoConfigure is not used

### ISSUE THREAD

**Issue by Divaite** [SeleniumHQ/selenium#5050]
## Meta
OS: Windows
Selenium Version: 3.7.0
Browser: Any

## Expected Behavior
Browser is starting and using .pac file from ProxyAutoConfigUrl  specified via Proxy settings.

## Actual Behavior
System.InvalidOperationException: unknown error: cannot parse capability: proxy
from unknown error: 'proxyAutoconfigUrl' must be a string
  (Driver info: chromedriver=2.33.506120 (e3e53437346286c0bc2d2dc9aa4915ba81d9023f),platform=Windows NT 6.1.7601 SP1 x86_64)
   at OpenQA.Selenium.Remote.RemoteWebDriver.UnpackAndThrowOnError(Response errorResponse)
   at OpenQA.Selenium.Remote.RemoteWebDriver.Execute(String driverCommandToExecute, Dictionary`2 parameters)
   at OpenQA.Selenium.Remote.RemoteWebDriver.StartSession(ICapabilities desiredCapabilities)
   at OpenQA.Selenium.Remote.RemoteWebDriver..ctor(ICommandExecutor commandExecutor, ICapabilities desiredCapabilities)

## Steps to reproduce
```
var proxy = new Proxy();
proxy.Kind = ProxyKind.ProxyAutoConfigure;
proxy.ProxyAutoConfigUrl = "http://localhost:8080/proxy.pac"; //anything here

var profile = new FirefoxProfile();

var options = new FirefoxOptions {Profile = profile, Proxy = proxy};
new FirefoxDriver(service, options);
```

Looks like something is lost, because I could not find any usage of **Proxy.ProxyAutoConfigUrl** property, especially in **Proxy.ToCapability()** function. Specified functionality works fine in Selenium 3.6.

**Comment by Divaite** [comment]
ping. could anybody check it at least? there is already version 3.8.0 and no news on this.

**Comment by jimevans** [comment]
This should be fixed in eaee33c. If there are other symptoms, please feel free to open a new issue.
